### ISSUE TITLE
convert a keras model and load it on pi3, can not work

### ISSUE THREAD

**Issue by dfybc** [tensorflow/tensorflow#49203]
I convert a keras h5 model exported by teachable maching image classfication  to a tflite quantized model.
and load the converted model on pi3(with tflite-runtime 2.1.0), such error happend:
  didn't find op for builtin opcode 'conv_2d' version '5'

also i install tensorflow 2.4.0-rc2 on pi3.
so i tried to load the converted model , no error happend,
but the prediction accuracy is not good at all.

when i load the tflite quantized model export by teachable maching on pi3, it worked very well.
any one know why? and how google do it?

### 1. System information
I convert the keras model to tflite quantized model on the following environment:
- OS Platform and Distribution ( Linux Ubuntu 16.04):
- TensorFlow installation (pip package or built from source): pip
- TensorFlow library (version): 2.4.0

### 2. The convert Code
import tensorflow as tf
import keras
model = keras.models.load_model('./keras_model/keras_model.h5')
converter = tf.lite.TFLiteConverter.from_keras_model(model)
converter.optimizations = [tf.lite.Optimize.DEFAULT]
tflite_model = converter.convert()
with open('model.tflite', 'wb') as f:
f.write(tflite_model)

**Comment by bhack** [comment]
> when i load the tflite quantized model export by teachable maching on pi3, it worked very well.

I don't understand what It is the configuration that It is "working well".

**Comment by tilakrayal** [comment]
@dfybc ,

In order to reproduce the issue reported here, could you please provide the complete code and the dataset  or colab link you are using. Thanks!

**Comment by tilakrayal** [comment]
@dfybc ,

Please let us know if your issue got resolved.Also please take a look at this issues.It helps.[link1](https://github.com/tensorflow/tensorflow/issues/42305),[link2](https://github.com/tensorflow/tensorflow/issues/46409),[link3](https://github.com/tensorflow/tensorflow/issues/43372).

Thanks!
